it/storage/README.md

```diff
@@ -0,0 +1,14 @@
+# Storage Stack
+
+This template is used for the storage stacks used for IT purposes
+
+## S3 Buckets
+
+There are many S3 buckets used as part of the CI/CD that are defined within CloudFormation templates, and don't need to be managed separately. A few buckets, though, must exist prior to launching any parts of this system, and must be configured correctly. The following describe each of those buckets:
```

Comment:
The buckets described by this template aren't prerequisites for CI or CD

Reply:
yeah, this was not ready for review, I was showing @debenedictis how to work with the infrastructure repo, and how to start a PR by opening it. I'll do some work on it with him, and hopefully then we'll be ready.